src/oaklib/selector.py

```diff
@@ -38,6 +38,22 @@
     "json-ld": "json-ld",
 }
 
+SCHEME_DICT = {
```

Comment:
good chance to look into #127, which uses `class-resolver` to accomplish things like this